katas/content/complex_arithmetic/complex_exponents/Solution.qs

```diff
@@ -0,0 +1,16 @@
+namespace Kata { 
+    open Microsoft.Quantum.Math;
+
+    operation ComplexExponent (x : Complex) : Complex {
+        
+        let e = 2.7182818284590452354;
```

Comment:
Math namespace has function `E()` for this constant - you're using it in verification, no reason not to teach the learner about that